### ISSUE TITLE
Custom routes issue with switchLocalePath

### ISSUE THREAD

**Issue by simoroma** [nuxt-modules/i18n#2033]
### Environment

"@nuxtjs/i18n": "^8.0.0-beta.10",

### Reproduction

I have the following pages:
- `terms.vue`
- `terms-fi.vue`

and my `nuxt.config.ts` has the following:
```
        customRoutes: 'config', // disable custom route with page components
        pages: {
            terms: {
                fi: '/terms-fi',
            },
        },
```
It works correctly when going from Finnish to English. Though when I am at `/fi/terms-fi` (the Finnish page) and I switch to english with `switchLocalePath` the routes becomes `terms-fi` and it should just be `terms`

### Describe the bug

Error in the route when switching language if custom pages are defined.

**Comment by riddla** [comment]
Probably because the name of your page (`terms-fi.vue`) is equal to the path you provide via the `pages` config (`/terms-fi`).

Why do you have two pages for terms? What do you want to achieve?

**Comment by simoroma** [comment]
Thanks for the reply. I am trying to get something similar to this: https://i18n.nuxtjs.org/custom-paths/ It is just more convenient to have two pages for terms because they have some paragraphs and I don't want to define many variables and get translations for each variable via `$t`.

I tried different variations of this. I still have the same issue with:
```
customRoutes: 'config', // disable custom route with page components
        pages: {
            terms: {
                en: '/terms',
                fi: '/terms-fi',
            },
        },
```

**Comment by riddla** [comment]
I don't think you can achieve that with nuxt-i18n (easily). The customs paths are meant to provide the paths to one single page.

It's actually quite common to have a lot of translated paragraphs (respectively variables) in pages. Maybe the tooling around vue-i18n over at https://kazupon.github.io/vue-i18n/guide/tooling.html can help you translate/work faster? I am using https://www.codeandweb.com/babeledit and https://marketplace.visualstudio.com/items?itemName=lokalise.i18n-ally to speed things up.

**Comment by simoroma** [comment]
Cool stuff. I might try to use them then.

I wanted to have two template files so I could send them to the website owner: eg `terms-en.vue` and `terms-fi.vue`. Then they would do the translations.

**Comment by riddla** [comment]
BabelEdit can read your vue-i18n language files and export them to a CSV file. Hand it over to your client and import it again when they are finished translating.
This is a quite convenient workflow, we have used them in several Nuxt (2) projects.

**Comment by BobbieGoede** [comment]
The custom/localized routes functionality is used to localize the path to the same page file/component. The content can be localized in any way you like but the usual first approach is using translation functions `$t()`, or by fetching translated content from a remote source. 

In your case you could still use the functionality provided by this module, but instead of having multiple page files for each language, you could display a component inside the page that contains the localized content for that language.

I will be closing this issue for now, since I think the use case is not a common one we do not intend to add functionality to support this.
